app/models/activity.rb

```diff
@@ -22,7 +22,7 @@ def project=(new_project); end
   # -- Scopes ---------------------------------------------------------------
 
   scope :latest, -> do
-    includes(:trackable).order("`activities`.`created_at` DESC").limit(10)
+    includes(:trackable).order(Arel.sql("`activities`.`created_at` DESC")).limit(10)
```

Comment:
The discussion around this fix was an interesting read but it made me wonder why we needed to fix this instance in the app, but no other instance of `.order("created_at DESC")` was throwing the same deprecation. Seems to be related specifically to the backticks which may be needed under certain circumstances but not in ours. As long as were dealing with column names `order("activities.created_at DESC")` or even `order("created_at DESC")` will do.

Reply:
Thanks. Removed backticks.